flyteidl/protos/flyteidl/service/admin.proto

```diff
@@ -372,20 +443,32 @@ service AdminService {
     // };
   }
 
-  // Updates an existing :ref:`ref_flyteidl.admin.Project` 
+  // Updates an existing :ref:`ref_flyteidl.admin.Project`
   // flyteidl.admin.Project should be passed but the domains property should be empty;
   // it will be ignored in the handler as domains cannot be updated via this API.
   rpc UpdateProject (flyteidl.admin.Project) returns (flyteidl.admin.ProjectUpdateResponse) {
     option (google.api.http) = {
       put: "/api/v1/projects/{id}"
       body: "*"
+      additional_bindings {
+        put: "/api/v1/projects/{id}"
+        body: "*"
+      }
+      additional_bindings {
+        put: "/api/v1/projects/org/{org}/{id}"
+        body: "*"
+      }
+      additional_bindings {
+        put: "/api/v1/projects/org/{org}/{id}"
+        body: "*"
+      }
```

Comment:
are these 2 sets of duplicates? or what am i missing?

Reply:
yes, sorry - thanks for the catch!